samples/nrf9160/cell_pos/src/main.c

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <modem/lte_lc.h>
+#include <net/cloud.h>
+#include <net/socket.h>
+#include <net/nrf_cloud_cell_pos.h>
+#include <dk_buttons_and_leds.h>
+#include <power/reboot.h>
+
+#include <logging/log.h>
+
+#define SERVICE_INFO_GPS \
+	"{\"state\":{\"reported\":{\"device\": {\"serviceInfo\":{\"ui\":[\"GPS\"]}}}}}"
+
+LOG_MODULE_REGISTER(scell, CONFIG_CELL_POS_SAMPLE_LOG_LEVEL);
+
+static struct cloud_backend *cloud_backend;
+static struct k_delayed_work scell_work;
+static struct k_delayed_work reboot_work;
+
+static void cloud_send_msg(void)
+{
+	int err;
+
+	LOG_DBG("Publishing message: %s", log_strdup(CONFIG_CLOUD_MESSAGE));
+
+	struct cloud_msg msg = {
```

Comment:
i don't think this message is needed for this sample.
perhaps use the button to submit another location request instead?

Reply:
Removed, add new button handling.